### PR TITLE
feat: add knownVulnerabilities to derivation metadata

### PR DESCRIPTION
**Pull request by fricklerhandwerk**
- use a JSON field, because it's a list of strings in Nixpkgs
  - m2m imposes additional burden on ingestion logic

### CODE REVIEW

**Comment by RaitoBezarius** [comment]
no, please just use an arrayfield from postgresql: https://docs.djangoproject.com/en/4.2/ref/contrib/postgres/fields/

**Comment by fricklerhandwerk** [comment]
Oh, are we hardcoding Postgres? Okay, makes things easier.

**Comment by RaitoBezarius** [comment]
Now that we are leaning towards full text search and various things, yes. SQLite was just for bootstrapping. :)

**Comment by fricklerhandwerk** [comment]
Rebased

File: src/website/shared/models/cve.py [hunk 1]
```diff
@@ -293,13 +295,13 @@ class NixpkgsIssue(models.Model):
     created = models.DateField(auto_now_add=True)
     code = models.CharField(max_length=len("NIXPKGS-YYYY-") + 19)
 
-    cve = models.ManyToManyField(CveRecord)
```
Comment (RaitoBezarius) [inline src/website/shared/models/cve.py:296]: I don't understand what is the value of this change, also it seems unrelated to that PR

**Comment by RaitoBezarius** [comment]
Also, we need the counterpart for the ingestion

File: src/website/shared/models/cve.py [hunk 2]
```diff
@@ -300,6 +302,7 @@ class NixpkgsIssue(models.Model):
         choices=IssueStatus.choices,
         default=IssueStatus.UNKNOWN,
     )
+    derivations = models.ManyToManyField(NixDerivation)
```
Comment (RaitoBezarius) [inline src/website/shared/models/cve.py:305]: what does it serve as a purpose?

**Review by RaitoBezarius — APPROVED** [review]
